README.md

```diff
@@ -102,6 +104,31 @@ class Task {
 }
 ```
 
+## Typed extras
+If you want to add static extra to all requests.
+
+```dart
+class MetaData extends TypedExtras {
+  final String id;
+  final String region;
+
+  const MetaData({required this.id, required region});
+}
+
+@MetaData(
+  id: '1234',
+  region: 'ng',
+)
+@GET("/get")
+Future<String> fetchData(@Extras() Map<String, dynamic> extras);
```

Comment:
I think you should remove the `extras` parameter, since it's not required for the example. So it doesn't end up confusing people.

Reply:
Absolutely